scanners/boostsecurityio/baseline/rules.yaml

```diff
@@ -0,0 +1,129 @@
+rules:
```

Comment:
Given that you boostsecurityio/scanner, maybe you should 
```
import:
- boostsecurityio/scanner
```

Instead of listing the rules... unless I missed something